modules/reducers/test/test_feature_selection.py

```diff
@@ -10,44 +10,32 @@
 
 
 class TestFeatureSelection(TestCase):
-    def test_variance_threshold_selector(self):
 
-        # Arrange
-        data, start, target = self.setup_data()
+    def setUp(self):
+        self.data = data_io.get_data(setup.get_datasets_path(), "winequality-red.csv")
+        self.target = 'quality'
+        self.start = self.data.shape
 
+    def test_variance_threshold_selector(self):
         # Act
-        reducer = scikit_feature_selectors.variance_threshold_selector(data, target)
+        reducer = scikit_feature_selectors.variance_threshold_selector(self.data, self.target)
         end = reducer.shape
 
         # Assert fewer columns
-        self.assertLess(end[1], start[1], msg="Failed to beat baseline")
+        self.assertLess(end[1], self.start[1], msg="Failed to beat baseline")
```

Comment:
You have the assertion in a single function instead of having it in multiple calls in some of the other test files. Would it make Code Climate happier to do that for all of these?